data/src/main/java/org/tiqr/data/util/extension/ContextExtensions.kt

```diff
@@ -0,0 +1,34 @@
+import android.content.ActivityNotFoundException
```

Comment:
You are missing the package declaration here, which makes your import not have the prefixes